src/dialects/abstract/query-interface.types.ts

```diff
@@ -1,10 +1,26 @@
 import type { QueryRawOptions, Sequelize } from '../../sequelize';
-import type { AbstractQueryGenerator, CreateSchemaQueryOptions } from './query-generator';
+import type { AbstractQueryGenerator, CreateSchemaQueryOptions, ListSchemasQueryOptions } from './query-generator';
+import type { CollateCharsetOptions } from './query-interface';
 
 export interface QueryInterfaceOptions {
   sequelize: Sequelize;
   queryGenerator: AbstractQueryGenerator;
 }
 
+export interface CreateDatabaseOptions extends CollateCharsetOptions, QueryRawOptions {
```

Comment:
Where in this PR is this used?

Reply:
I was originally going to migrate all schema and DB methods. I can remove this as the scope has changed, forgot about it